base/library/protected-annotation/src.rego

```diff
@@ -0,0 +1,17 @@
+package kubernetes.validating.annotaion
```

Comment:
typo

Reply:
I don't think it makes any practical difference, but the other policies in this repo have package names in the format:
```suggestion
package protectedannotation
```

Reply:
> I don't think it makes any practical difference
Indeed, AFAIK it's only used it to share rules